### DISCUSSION TITLE
Mir updated attachment nodes

### DISCUSSION THREAD

**Discussion by Toa-Sampa** [zer0Kerbal/MIR#57] [General]
[Mir Node fixes.zip](https://github.com/zer0Kerbal/MIR/files/11483043/Mir.Node.fixes.zip)

**Comment by zer0Kerbal** [comment]
added. Any more node adjustments @Toa-Sampa ?

**Comment by Toa-Sampa** [comment]
I will check once I download the new version

**Comment by zer0Kerbal** [comment]
@Toa-Sampa 

The part configs are reflecting your changes - so if they are off, then must be a scaling issue:
I know that rescaleFactor will affect the nodes (it auto scales them)... maybe scale has similar effects?


```
	scale = 0.95
	rescaleFactor = 1.0
	bulkheadProfiles = size1

	vesselType = Station
	CrewCapacity = 5
	INTERNAL { name = MIR-core }

	attachRules = 1,0,1,1,0
	node_stack_left = -0.60325, 3.81, 0.0, -0.95, 0.0, 0.0, 1, 1
	node_stack_right = 0.60325, 3.81, 0.0, 0.95, 0.0, 0.0, 1, 1
	node_stack_front = 0.0, 3.81, 0.60325, 0.0, 0.0, 0.95, 1, 1
	node_stack_rear = 0.0, 3.81, -0.60325, 0.0, 0.0, -0.95, 1, 1
	node_stack_top = 0.0, 4.31325, 0.0, 0.0, 0.95, 0.0, 1, 1
	node_stack_bottom = 0.0, -3.88125, 0.0, 0.0, -0.95, 0.0, 2, 1
	node_attach = 0.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0, 1
```

**Comment by zer0Kerbal** [comment]
@Toa-Sampa 

Yes, scale also affects in game vs. part.cfg node numbers

from the part.cfg
```
MODEL { model = BobCat/SovietPack/MIR/Assets/mir-kvant-solar }
	scale = 0.64
	rescaleFactor = 1.0
	bulkheadProfiles = srf

	attachRules = 0,1,0,0,1
	node_attach = 0.0, -0.20, 0.0, 0.0, -1.0, 0.0
```

and from ingame Node Helper export - (no changes)
```
node_attach = 0.0, -0.128, 0.0, 0.0, -0.64, 0.0, 1, 1
```

> -0.20 * 0.64 = -0.128

so have to divide the NH output by scale = 0.64 to reverse changes to get what needs to go into the part.cfg

> -0.128 / 0.64 = -0.20

_hopefully this makes sense_ ❓

**Comment by zer0Kerbal** [comment]
updated the three solar panels nodes.

**Comment by Toa-Sampa** [comment]
Alright.  Will check it out.  While I was waiting for news, I had set up
rss on my main, so, will revert those changes later and take a look
